components/forms/TransactionSigning.js

```diff
@@ -41,8 +40,29 @@ const TransactionSigning = (props) => {
           disableBalanceCheck: true,
         },
       };
+      const options = {
+        aminoTypes: new AminoTypes({
```

Comment:
With the change in https://github.com/cosmos/cosmjs/pull/1081 (0.27 -> 0.28), this overrides all types in `AminoTypes` instead of adding to them. Since this code is used for both message types (right?), we break the MsgSend with this.